app/views/sponsorships/_sponsorship.html.erb

```diff
@@ -0,0 +1,18 @@
+<div class="sponsorship">
+  <div class="row">
+    <div class="col-med-10 hidden-sm">
+      <h4>
+        <%= link_to sponsorship.organization.name, organization_path(sponsorship.organization) %>
+      </h4>
+      <% if sponsorship.message.present? %>
+        <p><%= sponsorship.message %></p>
+      <% end %>
+      </div>
```

Comment:
Took me a while to spot! 😄 It made the front-page sidebar appear at the bottom of the page